### ISSUE TITLE
As the Program Owner of another ED program, I would like to provide targeted CSVs for my program's audience that are always up to date.

### ISSUE THREAD

**Issue by LisaGee** [18F/college-choice#1338]
Michael Itzkowitz asked for.
One solution might be:  embed the Simple Query Builder and CSV creator/downloader onto his other ED page. He wants the ability for data analysts to download CSVs that include only the fields that make sense for this particular program.

Another option, which is really what he wants, is to enable him to put a button on his program page that says click here to get my Program's latest data and have a csv created real-time (that only includes the fields he wants) and be downloaded.

**Comment by saintsoup52** [comment]
Plus... this would greatly help schools with the financial aid shopping sheet compliance.

**Comment by LisaGee** [comment]
@saintsoup52 Can you help with more detail about how this type of thing would/could be used in the scenario you describe?

**Comment by LisaGee** [comment]
First, we would need to implement #165

**Comment by saintsoup52** [comment]
Non-sophisticated developers and researchers like myself would like to grab this data and use it but don't know how to extract flat files from an API>

I imagine this would work a lot like how the current IPEDS tool works... you select the data fields you want and you get a flat file on the back end.
